Library/Homebrew/cmd/pull.rb

```diff
@@ -516,7 +516,11 @@ def verify_bintray_published(formulae_names)
           opoo "Cannot publish bottle: Failed reading info for formula #{f.full_name}"
           next
         end
-        bottle_info = jinfo.bottle_info(jinfo.bottle_tags.first)
+        if f.tap.remote.include?("Linuxbrew") && jinfo.bottle_tags.include?("x86_64_linux")
```

Comment:
This incantation of `f.tap.remote.include?("Linuxbrew")` has come up a few times now. Would you like to factor out a `tap.linux?` function? It can be done in its own PR either before or after this PR as you see fit.